### DISCUSSION TITLE
Export custom detections and import to other manager

### DISCUSSION THREAD

**Discussion by security-companion** [Security-Onion-Solutions/securityonion#13937] [Ideas]
Hi,
I would like to suggest the feature of exporting custom detections and modifications to detections made via the "Detections"-site in the GUI and importing them into another manager that I have running independently from the first setup.
Could this be added to security-onion?
Greetings
security-companion

**Comment by InfosecGoon** [comment]
I will pass this suggestion along to the developers. Thanks!
